tcms/rpc/tests/test_bug.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+from mock import patch, MagicMock
+
+from django.conf import settings
+from django.core.cache import cache
+from django.test import override_settings
+
+from tcms.rpc.tests.utils import APITestCase
+
+if 'tcms.bugs.apps.AppConfig' not in settings.INSTALLED_APPS:
+    raise unittest.SkipTest('tcms.bugs is disabled')
+
+
+class TestBug(APITestCase):
+    def _fixture_setup(self):
+        super()._fixture_setup()
+        self.url = "http://some.url"
+        self.expected_result = {
+            'title': 'Bug from cache',
+            'description': 'This bug came from the Django cache'
+        }
```

Comment:
Both attributes cause pylint failures, the checker should be disabled in the test file, see other test files.